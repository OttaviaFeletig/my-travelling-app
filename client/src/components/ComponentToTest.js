import React from "react";

export default function ComponentToTest() {
  return (
    <div>
      <p>Test1</p>
      <p>add to test</p>
    </div>
  );
}
