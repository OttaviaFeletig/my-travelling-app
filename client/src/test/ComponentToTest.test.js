import React from "react";
import { shallow } from "enzyme";
import ComponentToTest from "../components/ComponentToTest";
// require("../setupTest");

it("ComponentToTest component snapshot", () => {
  const wrapper = shallow(<ComponentToTest />);
  expect(wrapper).toMatchSnapshot();
});
