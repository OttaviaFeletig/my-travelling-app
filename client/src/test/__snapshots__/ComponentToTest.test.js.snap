// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentToTest component snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ComponentToTest />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p>
          Test1
        </p>,
        <p>
          add to test
        </p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Test1",
        },
        "ref": null,
        "rendered": "Test1",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "add to test",
        },
        "ref": null,
        "rendered": "add to test",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p>
            Test1
          </p>,
          <p>
            add to test
          </p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Test1",
          },
          "ref": null,
          "rendered": "Test1",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "add to test",
          },
          "ref": null,
          "rendered": "add to test",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
